### PR TITLE
IP Guidance for projects

### PR DESCRIPTION
**Pull request by afrittoli**
As discussed during the [TOC WG on Jul 18th](https://docs.google.com/document/d/1uBHar55fTInWF9Li4t0lyG3tTC8BRLU0FfBfsgk_Jrs/edit#heading=h.3qtcpael34sy), we shall provide projects with guidance about intellectual property and licenses.

Fixes: #28

### CODE REVIEW

File: process/new_project_onboarding.md [hunk 1]
```diff
@@ -12,17 +12,25 @@ with a longer but reasonable grace period for legal actions like trademark and p
...
-
+* Domain Transfer - project-related domains should be transferred to the Linux Foundation via an issue created here https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/group/19
+* Github Repos - project-related repositories must be located in a neutral LF Github organization. The current GitHub organization owner must invite the user "thelinuxfoundation" (no quotes) as an owner.  GitHub’s documentation for how to do that is here: https://docs.github.com/en/organizations/managing-organization-settings/transferring-organization-ownership
+* Zoom account - projects may request a zoom sub-account for scheduling community meetings. Please submit a ticket to [CDF Helpdesk](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/group/19), identifying key community members who should have access.
 
 ## Legal
 
-  * Any registered marks for the project must be transferred to the Linux Foundation.
+* Licenses: the CDF only permits [OCI licenses][oci-licenses]. Projects may decide which OCI permitted license(s) to adopt for the source code and other content produced by project community.
```
Comment (fdegir) [inline process/new_project_onboarding.md:21]: This should be OSI Licenses and not OCI Licenses.

File: process/new_project_onboarding.md [hunk 2]
```diff
@@ -12,17 +12,25 @@ with a longer but reasonable grace period for legal actions like trademark and p
...
 
-  * A press release announcing the new project will be put out by CDF. Please identify key messaging and community members to provide quotes and email pr@cd.foundation
-  * Podcast - projects are welcome to do a kickoff episode on The Pipeline Podcast, more detail here: https://cd.foundation/podcast/
-  * CDF Landscape - add the project to the [CDF Landscape](https://github.com/cdfoundation/cdf-landscape) for better visibility
+* A press release announcing the new project will be put out by CDF. Please identify key messaging and community members to provide quotes and email pr@cd.foundation
+* Podcast - projects are welcome to do a kickoff episode on The Pipeline Podcast, more detail here: https://cd.foundation/podcast/
+* CDF Landscape - add the project to the [CDF Landscape](https://github.com/cdfoundation/cdf-landscape) for better visibility
+
+
+[oci-licenses]: https://opensource.org/licenses/
```
Comment (fdegir) [inline process/new_project_onboarding.md:35]: ditto.

This should be OSI Licenses and not OCI Licenses.

File: process/new_project_onboarding.md [hunk 3]
```diff
@@ -12,17 +12,25 @@ with a longer but reasonable grace period for legal actions like trademark and p
...
 
 ## Legal
 
-  * Any registered marks for the project must be transferred to the Linux Foundation.
+* Licenses: the CDF only permits [OCI licenses][oci-licenses]. Projects may decide which OCI permitted license(s) to adopt for the source code and other content produced by project community. 
+* Intellectual Property: projects may decide their preferred approach, typically enforcing a Developer Certificate of Origin (DCO) or a Contributor License Agreement (CLA). The Linux Foundation offers a CLA service via [LFX/EasyCLA][easycla] which projects may decide to adopt. Projects may also adopt.
+
+### Trademarks and Service marks
+
+* Any registered marks for the project must be transferred to the Linux Foundation.
```
Comment (fdegir) [inline process/new_project_onboarding.md:26]: An addition to this is that not all our project names are registered trademarks.

We had a chat with the LF Legal regarding this in the past and here is what @micmarti85 reminded me about common law trademark.

_A common law trademark provides protection for a symbol, logo, product name, or other words or marks that identify the source of goods or services before it is registered with the state or federal government. If someone began using a confusingly similar mark with similar goods/services, we could attempt to enforce those common law rights (e.g., cease and desist letter, law suit). But Registration in the U.S. and elsewhere get you additional, and frankly more useful rights. The registration could block other applications, we could utilize TM abuse policies on social media/GitHub, etc. These rights are a paid service and would need to be added to the CDF budget._

File: process/new_project_onboarding.md [hunk 4]
```diff
@@ -12,17 +12,26 @@ with a longer but reasonable grace period for legal actions like trademark and p
...
-   * Zoom account - projects may request a zoom sub-account for scheduling community meetings. Please submit a ticket to [CDF Helpdesk](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/group/19), identifying key community members who should have access.
-
+* Domain Transfer - project-related domains should be transferred to the Linux Foundation via an issue created here https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/group/19
+* Github Repos - project-related repositories must be located in a neutral LF Github organization. The current GitHub organization owner must invite the user "thelinuxfoundation" (no quotes) as an owner.  GitHub’s documentation for how to do that is here: https://docs.github.com/en/organizations/managing-organization-settings/transferring-organization-ownership
+* Zoom account - projects may request a zoom sub-account for scheduling community meetings. Please submit a ticket to [CDF Helpdesk](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/group/19), identifying key community members who should have access.
```
Comment (afrittoli) [inline process/new_project_onboarding.md:17]: No change, only linting

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

**Pull request merged**